### PR TITLE
Nuzhy/85542/refactor reset password page

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1 @@
-src/translations/*.json merge=theirs
+src/translations/*.json merge=ours
```
[the remark this answers is not in the document]
Comment (aswathy-deriv) [reply .gitattributes:1]: why the merge name is changed to ours from theirs

[the remark this answers is not in the document]
Comment (Nuzhy-Deriv) [reply .gitattributes:1]: this change is not from me. those changes are from upstream master

File: src/pages/reset-password/index.tsx [hunk 2]
```diff
@@ -56,6 +56,8 @@ const ResetPassword = () => {
         send({ verify_email: trimSpaces(values.email), type: 'reset_password' }, (response) => {
             actions.setSubmitting(false)
 
+            console.log(response)
```
[the remark this answers is not in the document]
Comment (Nuzhy-Deriv) [reply src/pages/reset-password/index.tsx:59]: sure

[the remark this answers is not in the document]
Comment (Nuzhy-Deriv) [reply src/pages/reset-password/index.tsx]: removed

**Review by niloofar-deriv — APPROVED** [review]
LGTM :+1: 
Just update your PR with the base branch ([niloo/89003/refactoring-translations-parent](https://github.com/niloofar-deriv/deriv-com/tree/niloo/89003/refactoring-translations-parent)) please :pray: 
The reason for that is we updated the key extractions configs [here](https://github.com/binary-com/deriv-com/pull/4117), since all of the merges to the project is blocked right now I added the same configs to this branch so by having this your crowdin would not update like this

**Pull request merged**